### PR TITLE
Updated the internal documention for the GDScript class.

### PR DESCRIPTION
**Pull request by NathanWarden**

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -257,13 +257,14 @@
...
 			<return type="float">
 			</return>
-			<argument index="0" name="a" type="float">
+			<argument index="0" name="from" type="float">
 			</argument>
-			<argument index="1" name="b" type="float">
+			<argument index="1" name="to" type="float">
 			</argument>
-			<argument index="2" name="c" type="float">
+			<argument index="2" name="percent" type="float">
```
Comment (not-surt) [inline doc/base/classes.xml]: "percent" suggests it should be in the 0-100 range.
I think 0.0-1.0 range would be better described as "weight" or "position".

Comment (NateWardawg) [inline doc/base/classes.xml]: I was struggling with what to call it, "weight" sounds good, will change it momentarily. Thanks!

Comment (okamstudio) [inline doc/base/classes.xml]: the argument names are generated, if you want to change this it' s better
to do it directly on the source code where the function is bound

On Fri, Feb 20, 2015 at 5:32 PM, surt <notifications@github.com> wrote:

> In doc/base/classes.xml
> <https://github.com/okamstudio/godot/pull/1407#discussion_r25100502>:
>
> >  			</argument>
> > -			<argument index="2" name="c" type="float">
> > +			<argument index="2" name="percent" type="float">
>
> "percent" suggests it should be in the 0-100 range.
> I think 0.0-1.0 range would be better described as "weight" or "position".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/okamstudio/godot/pull/1407/files#r25100502>.
>



-- 

*OkamStudio*

**Comment by NateWardawg** [comment]
Thanks, I had changed it there this morning too and forgot to commit it. It's been updated.

**Pull request merged**